app/lib/actions/katello/environment/publish_container_repositories.rb

```diff
@@ -1,24 +1,26 @@
 module Actions
   module Katello
     module Environment
-      class PublishRepositories < Actions::EntryAction
+      class PublishContainerRepositories < Actions::EntryAction
         middleware.use ::Actions::Middleware::RemoteAction
 
         input_format do
           param :id
           param :name
         end
 
-        def plan(env, options = {})
-          repositories = options[:content_type] ? env.repositories.with_type(options[:content_type]) : env.repositories
+        def plan(env)
+          repositories = env.repositories
```

Comment:
```suggestion
          repositories = env.repositories.with_type(Katello::Repository::DOCKER_TYPE)
```
We'd need something like above to limit to container repos with this refactoring.

Reply:
Forgot that bit, thanks :+1:

Reply:
Fixed in the latest commit with `.docker_type`.